spec/rspec/rails/configuration_spec.rb

```diff
@@ -256,6 +256,31 @@ def self.application; end
       expect(group.mailer_class).to be(a_mailer_class)
       expect(group.new).to be_a(RSpec::Rails::MailerExampleGroup)
     end
+
+    describe 'cleans test mailbox between each example in all rspec-rails example' do
+      class BaseMailer < ActionMailer::Base
```

Comment:
Can this be an anonymous class in a let?

Reply:
yep! Good idea

Reply:
Could you explain you prefer?

Done with https://github.com/rspec/rspec-rails/pull/2293/commits/368484b0fe4195edc6fe84dbd95719de3ce0eca4